creator-node/src/serviceRegistry.js

```diff
@@ -48,6 +48,9 @@ class ServiceRegistry {
       this.snapbackSM = new SnapbackSM(this.libs)
       await this.snapbackSM.init()
     }
+
+    // write all cnodes registered on chain here
+    await writeAllRegisteredCNodesToRedis(audiusLibs)
```

Comment:
this is only used in this content fix. since this is a background task that shouldn't block any core user flows i didnt want this to impact startup or anything else